docs/docs/reference/built-in-components/gatsby-plugin-image.md

```diff
@@ -0,0 +1,183 @@
+
+You can compare the layouts in the video below:
+
+<video controls autoplay loop>
+  <source type="video/mp4" src="./layouts.mp4" />
+  <p>Your browser does not support the video element.</p>
+</video>
+
+To set the layout, pass in the type to the layout prop. e.g.
+
+```jsx
+<StaticImage
+  src="./dino.png"
+  alt="A dinosaur"
+  // highlight-next-line
+  layout="fixed"
+/>
+```
+
+For a dynamic image, pass it to the resolver:
+
+```graphql
+dino {
+  childImageSharp {
+    # highlight-next-line
+    gatsbyImageData(layout: FIXED)
+  }
+}
+```
+
+### `width`/`height`
+
+_Layouts: fixed and constrained_
+
+Size props are optional in `GatsbyImage` and `StaticImage`. Because the images are processed at build time, the plugin knows the size of the source image and can add the correct width and height to the `<img>` tag, so it displays correctly with no layout jumping. However, if you want to change the display size you can use the size options to do this.
+
+For a `fixed` layout, these define the size of the image displayed on screen. For a `constrained` image these define the maximum size, as the image will scale down to fit smaller containers if needed.
+
+If you set just one of these, the source image is resized to that width or height while maintaining aspect ratio. If you include both then it is also cropped if needed to ensure it is that exact size.
+
+### `aspectRatio`
+
+_Layouts: fixed, constrained, and fullWidth_
+
+The `aspectRatio` prop forces an image to the specified aspect ratio, cropping if needed. The value is a number, but can be clearer to express as a fraction, e.g. `aspectRatio={16/9}`
+
+For `fixed` and `constrained` images, you can also optionally pass either a `width` or `height`, and it will use that to calculate the other dimension. For example, if you pass `width={800} aspectRatio={4/3}` then `height` will be set to the width divided by the aspect ratio: so `600`. Passing `1` as the aspectRatio will crop the image to a square. If you don't pass a width or height then it will use the source image's width.
+
+For `fullWidth` images you don't specify width or height as it resizes to fit the screen width. Passing `aspectRatio` will crop the image if needed, and the height will scale according to the width of the screen. For example, if you set the aspectRatio to `16/9` then when the image is displayed full width on a screen that is 1024px wide, the image will be 576 pixels high.
```

Comment:
Just as an idea: How about using 1280px and 720px? Maybe HD and it's 16:9 ratio is more common

Reply:
@LekoArts  Great idea. Thanks!